app/views/conversations/_form.html.erb

```diff
@@ -1,4 +1,11 @@
-<%= form_with(model: conversation, class: "contents") do |form| %>
+<% 
+submit_path = if conversation.new_record?
+  my_conversations_path
+else
+  edit_my_conversation_path(conversation)
+end
```

Comment:
This got weird. Rails routing magic failed here and maybe this alone is a sign that the idea is too far from happy-path to go forward.

Other options include:
- different ways to organize the partials, eg put the fields in a partial and not the form header
- not using partials -- my experience is the create and edit views rapidly diverge anyway